easybuild/easyconfigs/p/PLINK/PLINK-2.00a2.3LM-x86_64.eb

```diff
@@ -0,0 +1,24 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'PackedBinary'
+
+name = 'PLINK'
+version = '2.00a2.3LM-x86_64'
```

Comment:
What is LM?

Reply:
Good question. I think when I started to look into it the version was called LM but that changed after 9.4. I done some changes regarding versioning as well.